src/main/java/ru/svgogin/service/spark/service/SparkService.java

```diff
@@ -29,26 +28,37 @@ public SparkService(SparkRepositoryDb sparkRepositoryDb,
 
   public Iterable<CompanyDto> findAll() {
     Iterable<Company> companies = sparkRepositoryDb.findAll();
+    log.info("All companies were successfully returned.");
     return StreamSupport.stream(companies.spliterator(), false)
         .map(this::toDto)
         .collect(Collectors.toList());
   }
 
-  public Optional<CompanyDto> findByInn(String inn) {
-    return sparkRepositoryDb.findByInn(inn).map(this::toDto);
+  public CompanyDto findByInn(String inn) {
+    var companyDto = sparkRepositoryDb.findByInn(inn)
+        .map(this::toDto)
+        .orElseThrow(() -> new NoSuchEntityException("Company with inn " + inn + " doesn't exist"));
+    log.info("Company with inn "
+             + companyDto.getInn()
+             + " was successfully returned.");
```

Comment:
Why do you use three lines for this? This makes your code more complicated
```suggestion
    log.info("Company with inn " + companyDto.getInn() + " was successfully returned.");
```

Reply:
Sometimes I get a CheckStyle warning: "A line length is more than 100 signs"

Reply:
Done